cmd/operator-opamp-bridge/operator/client.go

```diff
@@ -156,14 +178,20 @@ func (c Client) ListInstances() ([]v1alpha1.OpenTelemetryCollector, error) {
 	ctx := context.Background()
 	result := v1alpha1.OpenTelemetryCollectorList{}
 	err := c.k8sClient.List(ctx, &result, client.MatchingLabels{
-		ResourceIdentifierKey: ResourceIdentifierValue,
+		ManagedLabelKey: c.name,
```

Comment:
i couldn't figure out how to do an `in` query here 🙃

Reply:
Like this, I believe:

```go
import (
    "k8s.io/apimachinery/pkg/labels"
    "k8s.io/apimachinery/pkg/selection"
)

labelSelector := labels.NewSelector()
requirement, err := labels.NewRequirement(ManagedLabelKey, selection.In, []string{c.name, "true"})
labelSelector.Add(*requirement)
k8sClient.List(ctx, &result, client.MatchingLabelsSelector{labelSelector})
```